pubspec.yaml

```diff
@@ -15,6 +15,7 @@ executables:
 dependencies:
   args: ^2.3.0
   build: ^2.1.0
+  build_runner: ^2.3.3
```

Comment:
Reason for this change? Doesn't look like there is anything within this PR that would warrant promoting build_runner from a dev dep to a regular dep

Reply:
Good catch, I was trying to fix a CI error and this seemed to resolve it, but I think it was something else that fixed it and this didn't get moved back.